documentation/en/source/pages/migration/migration-spec/migration-source.rst

```diff
@@ -31,3 +31,4 @@ The valid keys to use within a source are:
 * ``ssh-port`` (optional): an integer providing the ssh port of the running system (default: 22).
 * ``user`` (mandatory): a string providing the user login to authenticate to the live system.
 * ``password`` (optional): a string providing the user password to authenticate to the live system.
+* ``exclude`` (optional): a list of non-native files or directories to exclude during the deep scan of the live system.
```

Comment:
I'm worried regarding the term `deep scan`. Do we want to keep it?

Reply:
Asked to PO. As we have the same in scan run doc, it's ok for him